lib/ziti_tunneler_priv.h

```diff
@@ -21,6 +21,7 @@ typedef enum  {
 struct tunneler_io_ctx_s {
     tunneler_context    tnlr_ctx;
     const char *        service_name;
+    char                client[32];
```

Comment:
32 chars will be enough? :)

Reply:
32 is plenty. e.g.:

    tcp:123.456.789.012:123456

and it is populated with snprintf.